### PR TITLE
Fix VAD Termination

### PR DESCRIPTION
**Pull request by chevonc**
- Would cause the red mic to stay on the OS tray after ending call

### CODE REVIEW

File: ios/AudioInputController.m [hunk 1]
```diff
@@ -10,61 +10,76 @@
...
+
+        [audioSession setMode:AVAudioSessionModeVoiceChat error:nil];
+
+        NSLog(@"[WebRTCVad] set mode %d", ok);
+
+        // 5 second audio buffer hint
+        [audioSession setPreferredIOBufferDuration:5 error:nil];
+
+    } @catch (NSException *e) {
+        NSLog(@"[WebRTCVad]: session setup failed: %@", e.reason);
```
Comment (rkoval) [inline ios/AudioInputController.m:73]: would it be worthwhile to pass in an ErrorLogger of some sort to this module such that we can get more sophisticated logging (i.e., Sentry), or is this exception mostly noise due to permissions?

File: ios/AudioInputController.m [hunk 2]
```diff
@@ -146,13 +161,13 @@ - (OSStatus) _initializeAudioGraph {
     if (_checkError(status, "Couldn't set RemoteIO's render callback on bus 0")) {
         return status;
     }
-
```
Comment (rkoval) [inline ios/AudioInputController.m:149]: do these whitespace changes mean that you found a formatter for objective-c?

File: ios/AudioInputController.m (hunk 1 above)
Comment (rkoval) [reply ios/AudioInputController.m:73]: it seems like this module could at least benefit from some common logger that will append this prefix consistently

File: ios/AudioInputController.m (hunk 2 above)
Comment (chevonc) [reply ios/AudioInputController.m:149]: `CMD + A` then `CTRL + I` in XCode

File: ios/AudioInputController.m (hunk 1 above)
Comment (chevonc) [reply ios/AudioInputController.m:73]: Yeah, I agree, at least a common function. The tricky thing here, though, is we need to specify formatters for each arg, which changes based on arg type. ie - %d -> digit, %f -> decimal, float, %@ -> object, etc. Will think on this a bit and consolidate

**Comment by chevonc** [comment]
Gonna merge this @rkoval to get voice fix in for tonight's session

**Pull request merged**